### DISCUSSION TITLE
Sorry but how to play or delete downloaded musik on ios app

### DISCUSSION THREAD

**Discussion by witzker** [UnicornsOnLSD/finamp#371] [General]
Hi
Im totally confused
When i switch to Downloads in the menue albums are displayed
When i klick on one it opens and shows the songs
BUT
Nothing else
No delet no play

What have I done wrong
Pls. Help

**Comment by UnicornsOnLSD** [comment]
You play downloaded songs the same way you do non-downloaded songs. The player screen shows in the bottom left whether a song is being streamed or played via the download. I always thought that Plexamp's method was awkward 🙃

**Reply by witzker** [reply]
Sorry, I do not get it!?
In the Menu I go to Downloads
There are the downloaded albums displayed - Right?
And now I want to lay one of them!

I klick the album 
It opens and songs are displayed - Right?

But THAT'S IT!
I can do nothing by kicking the song 
nothing to do with this song is possible!

Is it clear what I mean?

**Reply by UnicornsOnLSD** [reply]
I've uploaded some screenshots that show the relevant screens. The downloads screen is for managing downloads (deleting them, seeing file sizes etc).

![mpv-shot0001](https://user-images.githubusercontent.com/44349936/209410450-34d87a80-81e8-4cae-a665-70d964ecfa85.jpg)

![mpv-shot0002](https://user-images.githubusercontent.com/44349936/209410453-97baf2a3-4cfe-4064-8a1b-6b0ff606ba0b.jpg)

![mpv-shot0003](https://user-images.githubusercontent.com/44349936/209410455-46a5efc5-5b5d-4ce8-94d4-5774563dfcae.jpg)

**Comment by witzker** [comment]
No!
Pls post a screenshot of Menue Download
My screen has header Downloads
![image](https://user-images.githubusercontent.com/4120266/209411297-96aa2e37-ac1f-4407-8880-cc24b734b48d.png)

I dot know how to post my iPhone screen to show yoe how it looks foe me

Maybe i have a wrong version!
Do not find version number in menue!
Maybe good Idea to put it in menue below Logo?

**Reply by UnicornsOnLSD** [reply]
You don't need to go into the download menu, just find what you want in the music screen, which is what shows when you open the app :)

**Reply by witzker** [reply]
@ You don't need to go into the download menu

Where else?
How to find downloaded albums for deleting them?

**Reply by UnicornsOnLSD** [reply]
The next release will add a delete button in the downloads screen, for now you'll have to look for the albums in the music screen and delete them (the download button on the album screen will be a delete button if the album is downloaded)

**Reply by witzker** [reply]
Will it also be possible to play downloaded albums from download menu then?
What do you think?

**Comment by witzker** [comment]
When will the next version be released?

**Reply by UnicornsOnLSD** [reply]
I could probably make a release next week, I don't really have a strict schedule

**Reply by witzker** [reply]
Great THX

**Reply by witzker** [reply]
Any news 01.01.23

**Reply by UnicornsOnLSD** [reply]
Ah sorry I forgot, I'll make a release today if I don't forget 🙃

**Reply by witzker** [reply]
Many THX
Btw will the new version be on AppStore to update the existing one?
If not how to get new Version?

**Reply by UnicornsOnLSD** [reply]
Ended up working on university stuff today, I'll make the release tomorrow sorry

It'll just update from the App Store like any other app

**Reply by UnicornsOnLSD** [reply]
The new release just got approved on iOS, it should show up as an update now (or within a few hours to a day)
